src/js/components/AppListComponent.jsx

```diff
@@ -394,6 +394,8 @@ var AppListComponent = React.createClass({
         state.fetchState !== States.STATE_LOADING
     });
 
+    var totalColSpan = 8;
```

Comment:
You're right, also my opinion.
That happened because it's derived from the element attribute called ```colSpan```.
I will rename it.